rpc/gnmi/gnmi-specification.md

```diff
@@ -1434,6 +1434,11 @@ established `Subscribe` RPC. The message contains the following fields:
 *   `extension` - a repeated field used to carry gNMI extensions, as per the
     description in [Section 2.7](#27-extensions-to-gnmi).
 
+`SubscribeResponse` messages with the `update` field set to a `Notification`
+message are not guaranteed to be sent in timestamp order. This allows congestion
```

Comment:
I would just leave this at the first sentence, and maybe say "are not required to be sent, and must not be assumed to be in timestamp order".

Reply:
Do you mean that a message set with `Notification` is not required, or that the timestamp itself is not required?

Reply:
Ok, PTAL, I changed to "MUST NOT assume that these **messages** will be received", since there can only be a single timestamp in each message.